testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.tests.js

```diff
@@ -4232,27 +4228,6 @@ QUnit.module('templates', baseModuleConfig, () => {
         setTemplateEngine('default');
     });
 
-    // T474695
-    QUnit.test('jsrender row template should works', function(assert) {
```

Comment:
Question.

Why are we deleting this test?
Don't we support ```jsrender``` anymore?

Reply:
We're on the way to stop supporting jsrender because it does not support CSP. This deleting is approved by Essentials team